lib/src/command/add.dart

```diff
@@ -389,13 +391,17 @@ class AddCommand extends PubCommand {
           packagePath, pubspecInformation ?? '^${resultPackage.version}');
     }
 
+    log.fine('Added ^${resultPackage.version} to "$dependencyKey"');
```

Comment:
This is not what happens if adding a git dependency... etc.. I'm fine with a dumber message that just says:
`Added "${package.name}" to $dependencyKey`.